### PR TITLE
Fixed case-sensitive paramer value from ALL to all

### PR DESCRIPTION
**Pull request by jhkimnew**
The value of --output-hashing is case sensitive. We have to use "all" not "ALL".

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -59,7 +59,7 @@ if ($pack) {
         ($outputHashingIndex+1 -ge $args.Count) -Or
         ($args[$outputHashingIndex+1].ToLower() -ne "all")
```
Comment (shhsu) [inline build.ps1]: can you remove this .ToLower() as well

Comment (jhkimnew) [reply build.ps1]: @shhsu done.

Comment (shhsu) [reply build.ps1]: actually, I just realized, `-ne` does not work here at all because it is not case sensitive, can you use `.equals()` instead

Comment (jhkimnew) [reply build.ps1]: Fixed.

**Review by shhsu — APPROVED** [review]

**Pull request merged**